plugins/kinet/README.md

```diff
@@ -2,12 +2,38 @@ KiNET Plugin
 ============
 
 This plugin creates a single device with multiple output ports. Each port
-represents a power supply. This plugin uses the V1 DMX-Out version of the
-KiNET protocol.
+represents a power supply. This plugin supports both the DMX Out (V1) and
+the Port Out (V2) modes of the KiNET protocol.
 
 
 ## Config file: `ola-kinet.conf`
 
-`power_supply = <ip>`  
+`power_supply = <ip>`
 The IP of the power supply to send to. You can communicate with more than
-one power supply by adding multiple `power_supply =` lines
+one power supply by adding multiple `power_supply =` lines.
+
+### Per Power Supply Settings
+
+`<ip>-mode = [dmxout|portout]`
+The mode of KiNET to send to the power supply. DMX Out is sometimes known as
+V1 and Port Out as V2.
+
+In general, it is highly recommended to use the Port Out mode if your power
+supply supports it. Only very old devices do not support Port Out mode, so
+it is recommended to try Port Out mode first. OLA defaults to DMX Out mode
+for backwards compatibility reasons. Note that some newer devices only
+support the Port Out protocol. On some devices it is also necessary to use
+the Port Out protocol in order to address all of the connected fixtures if
+the number of channels required exceeds one universe.
+
+Note that for the DMX Out mode the universe option on the power supply will
+be ignored, as this plugin only supports unicast DMX Out packets destined
+for the wildcard universe on each device. Instead, the universe for each
+device may be patched by assigning this output port to the intended universe
+in OLA.
+
+`<ip>-ports = <int>`
+The number of physical ports available on the power supply in Port Out mode.
+Each physical port will create an OLA port that may be assigned to any
+universe. This setting is ignored in DMX Out mode. The default and maximum
+number of ports per power supply supported by OLA is 16.
```

Comment:
```suggestion
universe. This setting is ignored in DMX Out mode. The default and maximum
number of ports per power supply supported is 16.
```

The external tool I used only allowed up to 16 ports, should I allow/generate a larger range then?

Reply:
Technically based on the fact that it is a whole byte it should be able to go up to 255, though I have never encountered one with more than 8 ports. I think a more reasonable default is 2 ports (most common across all CK devices a least). Most devices I have encountered are either 1, 2, 4, or 8 ports.

Reply:
> Technically based on the fact that it is a whole byte it should be able to go up to 255, though I have never encountered one with more than 8 ports.

Yeah, given the non-CK tool I tried was limited to 16, I assumed that might be a formal limit in the protocol.

> I think a more reasonable default is 2 ports (most common across all CK devices a least). Most devices I have encountered are either 1, 2, 4, or 8 ports.

My main reason for default to the max is you then don't need to change the setting, you can just avoid patching the non-existent ports, although the option is present should you want to tidy things up.

Ideally we'd detect and auto-configure per device but I've no idea if that's actually possible.

Reply:
OK, then let's stick with 16.

>Yeah, given the non-CK tool I tried was limited to 16, I assumed that might be a formal limit in the protocol.

I just checked QuickPlay Pro 2 and it also only lets you add a device with 16, so I guess that must be the maximum.